build.zig

```diff
@@ -12,7 +12,7 @@ pub const Language = enum {
     luau,
 };
 
-pub fn build(b: *Build) void {
+pub fn build(b: *Build) !void {
```

Comment:
I don't see any use of `try` so the `!` shouldn't be necessary anymore. Perhaps left over from your process of getting things working?

Reply:
yes, you are correct